tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
Where is used this configuration?

Reply:
It is necessary to test the app-ts template.
I was planning to add a new issue to use the new shared tsconfig for that template too.

Reply:
ok nice!